frontend/src/components/personal/customization/CustomizationForm.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useCallback, useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { Button, FormControl, FormGroup, FormControlLabel, Checkbox, Select, 
+  InputLabel, MenuItem, TextareaAutosize, Typography } from '@material-ui/core'
+
+import { successChangeUserCustomization } from '../../../actions/customizationActions'
+import { RootState } from '../../../reducers/index'
+import { useStyles } from './CustomizationForm.styles'
+import { FIVE } from '../../../constants/styles.values'
+import { MINUTES_QTY_TWO_CHARS_LONG, SIXTY_MINUTES_IN_HOUR } from '../../../constants/valuableNumbers'
+
+export default function PreferencesCustomizationForm (): React.ReactElement {
+  const dispatch = useDispatch()
+  
+  const custom = useSelector((state: RootState) => state.customizationReducer)
+
+  const [checkboxState, setCheckboxState] = useState({
+      sendEmailsWithNews: custom? custom.sendEmailsWithNews : true,
```

Comment:
Why don't we use formik+yup?

Reply:
corrected with adding Formik.
We dont use Yup here as there is no validation since its checkboxes and `About Me` field is optional.